app/main.py

```
@@ -52,21 +57,84 @@
+            return False
+            
+    return False
+
+# Initialize database with schema and tables
+async def initialize_database():
+    """
+    Initialize the database schema and seed data
+    """
     try:
         # Import models to ensure they're registered with Base before initialization
         from . import models
```

Ruff (0.8.2)

`.models` imported but unused

Remove unused import: `.models`

(F401)